apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/_components/Sidebar/ClientFilesTree/index.tsx

```diff
@@ -31,18 +30,16 @@ export default function ClientFilesTree({
   const isMerged = !!commit.mergedAt
   const { project } = useCurrentProject()
   const documentPath = currentDocument?.path
-  const selectedSegment = useSelectedLayoutSegment() as DocumentRoutes | null
   const navigateToDocument = useCallback(
     (documentUuid: string) => {
       const documentDetails = ROUTES.projects
         .detail({ id: project.id })
         .commits.detail({ uuid: isHead ? HEAD_COMMIT : commit.uuid })
         .documents.detail({ uuid: documentUuid })
 
-      if (!selectedSegment) return router.push(documentDetails.root)
-      router.push(documentDetails[selectedSegment].root)
+      return router.push(documentDetails.root)
     },
-    [selectedSegment, project.id, commit.uuid, isHead],
+    [project.id, commit.uuid, isHead],
   )
```

Comment:
Isn't this more confusing? If I'm in the logs tab of a document, when clicking on another document imo I expect to see the logs of that document.

Reply:
i've seen recordings of users getting more confused about switching documents and not seeing the editor when they are in the logs tab, specially when the logs tab is empty and nothing changes visually